docs/en/about-appium/platform-support.md

```diff
@@ -96,3 +96,7 @@ This section lists drivers that are supported by vendors and community.
 ### Roku Support
 
 - The [Roku Driver](https://github.com/cbuelvasc/appium-roku-driver)
+
+### WebOS Support
+
+- The [WebOS Driver](https://github.com/cbuelvasc/appium-webos-driver)
```

Comment:
Oh, I noticed both drivers are now private(?).
Do you have any places users can report issues etc and documentation refer to? @cbuelvasc 

If no, it is helpful for users to leave notice they are private/commercial only unlike other drivers in this list which are available publicly.